samples/batch/transcription-enabled-storage/CreateTranscription/CreateTranscription.cs

```diff
@@ -0,0 +1,295 @@
+            log.LogInformation($"Received message: SequenceNumber:{message.SystemProperties.SequenceNumber} Body:{Encoding.UTF8.GetString(message.Body)}");
+
+            var location = TranscribeMessage(message, log).Result;
+            log.LogInformation("Request Created, Guid: " + location);
+        }
+
+        internal static async Task<string> TranscribeMessage(Message message, ILogger log)
+        {
+            var transcriptionCreationTime = DateTime.UtcNow;
+            var subscriptionKey = CreateTranscriptionEnvironmentVariables.AzureSpeechServicesKey;
+            var subscriptionRegion = CreateTranscriptionEnvironmentVariables.AzureSpeechServicesRegion;
+            var locationUri = new Uri($"https://{subscriptionRegion}.api.cognitive.microsoft.com");
+            var subscription = new Subscription(subscriptionKey, locationUri);
+
+            var messageBody = Encoding.UTF8.GetString(message.Body);
+            var busMessage = JsonConvert.DeserializeObject<ServiceBusMessage>(messageBody);
+
+            var dataUri = busMessage.Data.Url;
+            var modelIds = new List<Guid>();
+
+            if (busMessage.EventType.Contains("BlobCreate", StringComparison.OrdinalIgnoreCase))
+            {
+                var audioFileUrl = await StorageUtilities.CreateSASAsync(CreateTranscriptionEnvironmentVariables.AzureWebJobsStorage, dataUri.AbsoluteUri, log).ConfigureAwait(false);
+                var containerName = StorageUtilities.GetContainerNameFromPath(dataUri.AbsoluteUri);
+                var audioFileName = StorageUtilities.GetFileNameFromPath(dataUri.AbsoluteUri);
+                var audioInputContainer = CreateTranscriptionEnvironmentVariables.AudioInputContainer;
+
+                if (StorageUtilities.IsItAudio(audioFileName) && !string.IsNullOrEmpty(containerName) && containerName.Equals(audioInputContainer, StringComparison.Ordinal))
+                {
+                    log.LogInformation($"Create request for audio file {audioFileName}");
+
+                    var jobName = StorageUtilities.GetFileNameWithoutExtension(audioFileName);
+
+                    var primaryLocale = CreateTranscriptionEnvironmentVariables.Locale;
+                    primaryLocale = primaryLocale.Split('|')[0].Trim();
+                    var locale = primaryLocale;
+
+                    string locationString;
+                    byte[] byteArray = null;
+
+                    try
+                    {
+                        var secondaryLocale = CreateTranscriptionEnvironmentVariables.SecondaryLocale;
+
+                        if (!string.IsNullOrEmpty(secondaryLocale) &&
+                            !secondaryLocale.Equals("None", StringComparison.OrdinalIgnoreCase) &&
+                            !secondaryLocale.Equals(primaryLocale, StringComparison.OrdinalIgnoreCase))
+                        {
+                            secondaryLocale = secondaryLocale.Split('|')[0].Trim();
```

Comment:
It seems be more common to  use "," to separate locales.

Reply:
this is not between locales, but needed if the ARM template is used for deployment. In the ARM template, the locales are provided like this "en-US | English (United States)".